wfe/wfe.go

```diff
@@ -327,6 +332,12 @@ func (wfe *WebFrontEndImpl) Directory(ctx context.Context, logEvent *requestEven
 		"new-cert":    newCertPath,
 		"revoke-cert": revokeCertPath,
 	}
+	if features.Enabled(features.AllowKeyRollover) && !strings.HasPrefix(request.UserAgent(), "LetsEncryptPythonClient") {
```

Comment:
I don't think its worth blocking this PR on, but we should probably mention the selective directory entry behaviour in the  divergences documentation. It might be surprising to someone at some point down the road.